scapy/contrib/bgp.py

```diff
@@ -99,20 +99,30 @@ class BGPFieldIPv4(Field):
     """
     IPv4 Field (CIDR)
     """
-
+    alen = 4
+    afam = socket.AF_INET
     def mask2iplen(self, mask):
         """Get the IP field mask length (in bytes)."""
         return (mask + 7) // 8
 
     def h2i(self, pkt, h):
-        """x.x.x.x/y to "internal" representation."""
+        """
+        x.x.x.x/y to "internal" representation.
+        However, expect 0/0 and change that to 0.0.0.0/0 internally
+        """
+        if h == "0/0":
```

Comment:
Other notation are also valid. You shoukd use `inet_aton()` to check the values:
`socket.inet_aton("0") == socket.inet_aton("0.0.0.0")`